src/databricks/labs/remorph/cli.py

```diff
@@ -28,7 +28,7 @@ def transpile(
     w: WorkspaceClient,
     source: str,
     input_sql: str,
-    output_folder: str,
+    output_folder: str | None,
```

Comment:
`default_config.output_folder` is of type "str | None": 
Error: Incompatible types in assignment expression has type "str | None", variable has type "str"
Fix: updated the type of `output_folder`